terraform/eks_arm64_cluster_setup/main.tf

```diff
@@ -0,0 +1,123 @@
+# ------------------------------------------------------------------------
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+# -------------------------------------------------------------------------
+
+terraform {
+  required_version = "=1.1.6"
+  required_providers {
+    kubectl = {
+      source  = "gavinbunney/kubectl"
+      version = ">= 1.7.0"
+    }
+  }
+  backend "s3" {
+    bucket = "adot-op-cluster-terraform-statefile"
+    key    = "eks_arm64_cluster_setup/terraform.tfstate"
+    region = "us-west-2"
```

Comment:
should this be `var.region`

Reply:
No. Backend does not accept variables according to https://github.com/hashicorp/terraform/issues/13022.